sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -1557,6 +1557,17 @@ class PlanResolutionSuite extends AnalysisTest {
     checkFailure("testcat.tab", "foo")
   }
 
+  test("SPARK-32356: forbid null type in create view") {
+    val sql1 = "create view v as select null as c"
+    val sql2 = "alter view v as select null as c"
+    Seq(sql1, sql2).foreach { sql =>
+      val msg = intercept[AnalysisException] {
+        parseAndResolve(sql)
+      }.getMessage
+      assert(msg.contains(s"Cannot create tables with ${NullType.simpleString} type."))
```

Comment:
shall we update the error message to be `tables/views`?

Reply:
updated